pkg/runTest.py

```diff
@@ -4,11 +4,13 @@
 
 import logging
 from fmatch.matcher import Matcher
+from pkg.algorithmFactory import AlgorithmFactory
 from pkg.logrus import SingletonLogger
-from pkg.utils import run_hunter_analyze, get_es_url, process_test
+from pkg.utils import get_es_url, process_test
+from pkg.types import OptionMap
+import pkg.constants as cnsts
 
-
-def run(**kwargs):
```

Comment:
Is there a specific reason we have replace **kwargs with the OptionMap?

Reply:
The reason why we are going ahead with OptionMap structure instead of kwargs is, some of the options are being used across other functions such as the anomaly window for anomaly detection. For cleaner code it was better to call OptionMap instead of passing kwargs throughout.